README.md

```diff
@@ -0,0 +1,34 @@
+# Shuttle deploy action
+
+Deploys current project to [shuttle.rs](https://www.shuttle.rs)
+
+Will checkout the repository if not already checked out
+
+### Inputs
+
+`shuttle-deploy-key`, the key found at https://www.shuttle.rs/login
+
+`working-directory`, the folder which includes the project, **defaults to `"."`**
+
+### Outputs
+
+None
+
+### Example usage
+
+```yml
+name: Shuttle deploy
+
+on:
+  push:
+    branches:
+      - "main"
+
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: shuttle-hq/shuttle-deploy-action@main

```

Comment:
```suggestion
      - uses: shuttle-hq/deploy-action@main
```
I not sure if this change goes against "best practises" for actions :man_shrugging:

Reply:
Yep that makes it simpler and looking at other action names seems to be fine. The action reference mirrors the name of the repository, so if you want to change the repository name I will match it in the example in the readme 👍

Reply:
Also noted that is that I have it targeting `@main` here but this repositories default branch is `master` so I need to change it or vice versa

Reply:
Ok, @christoshadjiaslanis please rename the repo and the default branch to `main` :smile:

Reply:
@kaleidawave `master` has been renamed to `main`. You may need to run the following eventually:
```
git branch -m master main
git fetch origin
git branch -u origin/main main
git remote set-head origin -a
```